packages/wdio-crossbrowsertesting-service/src/launcher.ts

```diff
@@ -7,7 +7,12 @@ import logger from '@wdio/logger'
 const log = logger('@wdio/crossbrowsertesting-service')
 
 export default class CrossBrowserTestingLauncher {
-    constructor (options, caps, config) {
+    options: WebdriverIO.ServiceOption;
+    config: WebdriverIO.Config;
+    tunnel: boolean = false;
+    cbtTunnelOpts: CBTConfigInterface = undefined as any;
```

Comment:
Let's follow the convention and define these all as `private` starting with an `_`

```suggestion
    private _options: WebdriverIO.ServiceOption
    private _config: WebdriverIO.Config
    private _tunnel: boolean = false
    private _cbtTunnelOpts: CBTConfigInterface
```

Reply:
Done